dwave/system/composites/linear_ancilla.py

```diff
@@ -0,0 +1,214 @@
+# coding: utf-8
+# Copyright 2018 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+"""Composite to implement linear coefficients with ancilla qubits biased with flux-bias offsets.
+"""
+
+from collections import defaultdict
+import numbers
+from typing import Sequence, Mapping, Any
+
+import numpy as np
+
+from dimod.decorators import nonblocking_sample_method
+import dimod
```

Comment:
```suggestion
import numbers

from collections import defaultdict
from typing import Sequence, Mapping, Any

import dimod
import numpy as np

from dimod.decorators import nonblocking_sample_method
```

Also, IMO it makes for a less cluttered namespace to just `import typing` and then use `typing.Sequence` etc in the code. But happy either way.